locations/spiders/tanger_outlet.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from locations.items import GeojsonPointItem
+
+class TangerOutletSpider(scrapy.Spider):
+	name = "tanger_outlet"
```

Comment:
Can you switch to four spaces to match the rest of the repo please?

Reply:
Yep, will do that!